README.md

```diff
@@ -1,66 +1,123 @@
-
-1.  **Set up the environment.**  
-
-        pip install -r requirements.txt
-        sudo -u postgre createuser -s $USER
-        sudo -u postgre createdb kcsrv
+Ingredients:
+---  
+* Python 3
+* ```virtualenv```
+* PostgreSQL (+ development headers)
+
+Cooking Instructions:
+---  
+1.  **Set up the environment.**
+
+    	git clone http://github.com/kancolletool/kcsrv.git kcsrv
+    	cd kcsrv
+    	virtualenv -p /usr/bin/python3 .
+    	source ./bin/activate
+        
+    Now, enter the project's Virtual Environment. This contraption allows you to install packages locally, without conflicting with system packages. It's activated per-terminal, and must be activated before interacting with the project in any way:
+  
+        # Not a typo; `.` is an alias for `source`  
+        . bin/activate
+        
+    If you wish to exit it again, simply run:
+
+        deactivate
+        
+1. **Install dependencies.**  
+    Inside the virtualenv, simply run:
+
+    	pip install -r requirements.txt
+
+    If you ever get issues with things not being found, try re-running this and seeing what happens.
+    
+1. **Set up the database***  
+    First, make sure you have a database user for the account you plan on running the server as. Postgres can automatically authorize as the user a process is running as, which means there's no need to put passwords in configuration files.
 
+        sudo -u postgres createuser --superuser $USER
+        
+    Then, create a database for kcsrv to use:
+        createdb kcsrv  
 
-1.  **Create the database.**  
-    Repeat this to upgrade it later.
+    Change postgres to what user the server installs as. On Debian/Ubuntu it's postgres.
+    And fill it with data - repeat this after every upgrade:
+    
+        ./manage.py db upgrade   # Run migrations
+        ./manage.py db update_db # Imports ship data into the database
+        
+    First Boot? You'll need to set up a few things.
     
-        ./manage.py db upgrade
         ./manage.py setup
-        ./manage.py update_db
+
+1. **Download the DMM assets.**  
+    This will take a while... you may want to re-run it after game patches, to stay up-to-date. At this point, grab a coffee.
+
+    Please note that this won't pull new ships automatically, you'll need to feed the server a new api_start2.json every time DMM introduces more ships to collect. Gotta ship 'em all!
+
+        ./manage.py dlassets
 
 1.  **Create an account for yourself.**  
     Be sure to give yourself the `admin` and `staff` roles!
-    
-        ./manage.py user create
-
+    Depending on your terminal, you may need to put the email address in quotation marks.
 
+        ./manage.py user create username user@localhost
+    
+    NOTE: Your terminal may require you to escape the email address with quotes. Some terminals don't worry about it, some older ones may throw a error. If in doubt, enclose it with quotes to be on the safe side
```

Comment:
I'll believe this when you can find me a terminal that cares about @-signs, because that's not even remotely a special character. In fact, entering email addresses as parameters has been done since the invention of email (`sendmail user@example.com`), which makes it even more implausible.